### PR TITLE
Add base template type

### PR DESCRIPTION
**Pull request by SantiagoRodH**

### CODE REVIEW

File: ui.apps/src/main/content/jcr_root/apps/aem-bootcamp/components/page/customfooterlibs.html [hunk 1]
```diff
@@ -13,7 +13,13 @@
     See the License for the specific language governing permissions and
     limitations under the License.
 */-->
+
```
Comment (lflondonol92) [inline ui.apps/src/main/content/jcr_root/apps/aem-bootcamp/components/page/customfooterlibs.html]: Remove the commented code

File: ui.apps/src/main/content/jcr_root/apps/aem-bootcamp/components/page/customheaderlibs.html [hunk 2]
```diff
@@ -13,8 +13,16 @@
     See the License for the specific language governing permissions and
     limitations under the License.
 */-->
+
```
Comment (lflondonol92) [inline ui.apps/src/main/content/jcr_root/apps/aem-bootcamp/components/page/customheaderlibs.html]: Remove commented code

File: ui.apps/src/main/content/jcr_root/apps/aem-bootcamp/components/page/customheaderlibs.html [hunk 3]
```diff
@@ -13,8 +13,16 @@
...
+<!--/*
 <sly data-sly-use.clientlib="core/wcm/components/commons/v1/templates/clientlib.html">
     <sly data-sly-call="${clientlib.css @ categories='aem-bootcamp.base'}"/>
 </sly>
```
Comment (lflondonol92) [inline ui.apps/src/main/content/jcr_root/apps/aem-bootcamp/components/page/customheaderlibs.html]: Remove commented code

File: ui.apps/src/main/content/jcr_root/apps/aem-bootcamp/components/page/customheaderlibs.html [hunk 4]
```diff
@@ -13,8 +13,16 @@
...
 
+
```
Comment (lflondonol92) [inline ui.apps/src/main/content/jcr_root/apps/aem-bootcamp/components/page/customheaderlibs.html]: Remove commented code

File: ui.apps/src/main/content/jcr_root/apps/aem-bootcamp/components/page/customfooterlibs.html [hunk 5]
```diff
@@ -13,7 +13,13 @@
...
+<!--/*
```
Comment (valMoncadaS) [inline ui.apps/src/main/content/jcr_root/apps/aem-bootcamp/components/page/customfooterlibs.html]: es necesario dejar este comentario?

File: ui.content/src/main/content/jcr_root/conf/aem-bootcamp/settings/wcm/template-types/base-template/initial/.content.xml [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+<jcr:root xmlns:jcr="http://www.jcp.org/jcr/1.0" xmlns:nt="http://www.jcp.org/jcr/nt/1.0" xmlns:cq="http://www.day.com/jcr/cq/1.0" xmlns:sling="http://sling.apache.org/jcr/sling/1.0"
+    jcr:primaryType="cq:Page">
+    <jcr:content
+        jcr:primaryType="cq:PageContent"
+        sling:resourceType="aem-bootcamp/components/page">
+        <root
+            jcr:primaryType="nt:unstructured"
+            sling:resourceType="aem-bootcamp/components/container"
+            layout="responsiveGrid">
+            <container
```
Comment (lflondonol92) [inline ui.content/src/main/content/jcr_root/conf/aem-bootcamp/settings/wcm/template-types/base-template/initial/.content.xml:17]: The initial values for the XFs are missing

**Review by lflondonol92 — COMMENTED** [review]
Please resolve the comments

**Review by valMoncadaS — APPROVED** [review]
Looks good for me. Approved.

**Review by lflondonol92 — APPROVED** [review]
Approved!

**Pull request merged**